src/docs/cursor-join.dox

```diff
@@ -21,9 +19,7 @@ satisfy at least one of the joined comparisons.  More complex joins can be
 composed by specifying another join cursor as the reference cursor in a join
 call.
 
-Here is an example using these concepts to show a conjunction of a disjunction:
-
-@snippet ex_schema.c Complex join cursors
+Complex join cursors, such as a conjunction of a disjunction:
```

Comment:
I assume the doc removals are just the minimum needed to get the docs compiling now that the examples are removed?

Reply:
Yes. The full doc removal is in wt-12488.